### PR TITLE
Customize password reset email

### LINKED ISSUE #4933: Localizable strings missing for password reset emails
Reset password email is partially translated, the other english lines are not in the localization file (even the words like `Hello` or `Thank you for using`...

Reset Password email

**I would recommend to check other email templates you added for pootle**... just checked the confirmation pootle email and it is complete, but i do not understand why you add `<br></br>` to break the lines of the text

Confirmation Pootle email

### PR DESCRIPTION
**Pull request by unho**
This allows to localize it within Pootle.

Fixes #4933.

### CODE REVIEW

**Comment by phlax** [comment]
not tested, but lgtm, nice one @unho

**Comment by unho** [comment]
Actually I am going to reword this a bit.

File: pootle/templates/account/email/password_reset_key_message.txt [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{% load i18n %}
+{% filter wordwrap:70 %}{% trans "Your user account's password has been requested to be reset." %}{% endfilter %}
```
Comment (dwaynebailey) [inline pootle/templates/account/email/password_reset_key_message.txt]: How about:

"We've received a request to reset your password."

**Comment by dwaynebailey** [comment]
Wording suggestion.  Otherwise gtm

File: pootle/templates/account/email/password_reset_key_message.txt (hunk 1 above)
Comment (unho) [inline pootle/templates/account/email/password_reset_key_message.txt]: Changing.

**Pull request merged**